spyder/widgets/findinfiles.py

```diff
@@ -533,161 +485,125 @@ def __init__(self, parent):
         self.data = None
         self.set_title('')
         self.root_items = None
-        
+        self.setSortingEnabled(True)
+        self.header().setSectionsClickable(True)
+        # self.setHeaderLabel(_("Filename"))
+
     def activated(self, item):
         """Double-click event"""
         itemdata = self.data.get(id(self.currentItem()))
         if itemdata is not None:
-            filename, lineno = itemdata
+            filename, lineno, colno = itemdata
             self.parent().edit_goto.emit(filename, lineno, self.search_text)
 
     def clicked(self, item):
         """Click event"""
         self.activated(item)
-        
-    def set_results(self, search_text, results, pathlist, nb,
-                    error_flag, completed):
-        self.search_text = search_text
-        self.results = results
-        self.pathlist = pathlist
-        self.nb = nb
-        self.error_flag = error_flag
-        self.completed = completed
-        self.refresh()
-        if not self.error_flag and self.nb:
-            self.restore()
-        
-    def refresh(self):
-        """
-        Refreshing search results panel
-        """
-        title = "'%s' - " % self.search_text
-        if self.results is None:
-            text = _('Search canceled')
-        else:
-            nb_files = len(self.results)
-            if nb_files == 0:
-                text = _('String not found')
-            else:
-                text_matches = _('matches in')
-                text_files = _('file')
-                if nb_files > 1:
-                    text_files += 's'
-                text = "%d %s %d %s" % (self.nb, text_matches,
-                                        nb_files, text_files)
-        if self.error_flag:
-            text += ' (' + self.error_flag + ')'
-        elif self.results is not None and not self.completed:
-            text += ' (' + _('interrupted') + ')'
-        self.set_title(title+text)
+
+    def init(self, search_text):

```

Comment:
This is a bad name, please use a better one (I mean, there's already `__init__` and `initialize`)